test/ipynb/mpl/test_circuit_matplotlib_drawer.py

```diff
@@ -270,17 +270,17 @@ def test_pauli_clifford(self):
         self.circuit_drawer(circuit, filename="pauli_clifford.png")
 
     def test_u_gates(self):
```

Comment:
I think this test can be removed. We already test U, Phase, CU, and CPhase gates somewhere else, right?

Reply:
The only one not covered elsewhere is the CU and I think there should be one since it has 4 params instead of 3. I can eliminate the u_gates.png and replace one of the CY's in ctrl_labels.png with a CU.